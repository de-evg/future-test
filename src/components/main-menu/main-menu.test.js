import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';
import MainMenu from './main-menu';
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({adapter: new Adapter()});

it(`Render MainMenu correctly`, () => {
  const wrapper = shallow(<MainMenu />);
  const tree = shallowToJson(wrapper);
  expect(tree).toMatchSnapshot();
});