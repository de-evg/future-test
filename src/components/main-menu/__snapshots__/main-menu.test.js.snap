// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MainMenu correctly 1`] = `
<nav
  className="main-menu"
>
  <Connect(ShowAllBtn) />
  <Connect(ShowFavoriteBtn) />
  <Connect(ShowSearchFormBtn) />
  <Connect(ShowAddFormBtn) />
</nav>
`;
